src/main/java/io/gatling/mojo/GatlingMojo.java

```diff
@@ -311,6 +313,13 @@ private void saveListOfNewRunDirectories() throws IOException {
           writer.write(directory.getName() + System.lineSeparator());
         }
       }
+      writeExceptionIfExists(writer, exception);
+    }
+  }
+
+  private void writeExceptionIfExists(BufferedWriter writer, Exception exception) throws IOException {
+    if (exception != null) {
+      writer.write(LAST_RUN_FILE_ERROR_LINE + " : " + exception.getClass() + " : " + exception.getMessage() + System.lineSeparator());
```

Comment:
`exception.getMessage()` can be null, so what you're doing will concatenate the "null" string.
It would be better to:
* use a StringBuilder so you can only append ` : ` and message when message is not null.
* also log the causes recursively, the valuable information might not be in the original Exception.